Okta.AspNetCore/OpenIdConnectOptionsHelper.cs

```diff
@@ -55,6 +56,21 @@ public static void ConfigureOpenIdConnectOptions(OktaMvcOptions oktaMvcOptions,
             };
 
             oidcOptions.Events.OnRedirectToIdentityProvider = events.OnRedirectToIdentityProvider;
+
+            if (oktaMvcOptions.OnTokenValidated != null)
+            {
+                oidcOptions.Events.OnTokenValidated = oktaMvcOptions.OnTokenValidated;
```

Comment:
Question: should we also expose `Events` so people can set their own events?

Reply:
In Asp.Net Core it shouldn't be a problem as you can concatenate multiple handlers for the same events. However, this is not possible in Asp.Net with its equivalent `Notifications` where we already have our own internal events implementations such as `AuthorizationCodeReceived` or `RedirectToIdentityProvider`. If we want to be as consistent as possible between both libs I wouldn't do that for now.

Reply:
Makes sense 👍